journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedJournal.scala

```diff
@@ -150,18 +150,6 @@ object ReplicatedJournal {
     def delete(topic: Topic, latency: FiniteDuration): F[Unit]
 
     def purge(topic: Topic, latency: FiniteDuration): F[Unit]
-
-    // TODO MR remove with next major release
```

Comment:
Are we doing major release already? Or the comment is wrong?

Reply:
wrong comment - at the time of writing I didn't have fully worked out the plan for rollout

Reply:
we could have left them deprecated for sake of binary compatibility, but all of them are our "internal API", which is public only because of the way, how Scala generates Java bytecode.